common.gradle

```diff
@@ -48,8 +48,13 @@ project.ext.getCustomProperties = {
 
     def path = 'pwd'.execute().text.trim()
     def keystore = path + "/build/dist/" + PROJECT_TAG + "/keystore/"
-    props.setProperty("SSLTRUSTSTORE", keystore + "cacerts")
 
+    def keystorePrefix = keystore.replace('/', '\\/').replace('.', '\\.')
+    props.setProperty("SSLKEYSTOREWORKER", keystorePrefix + "xwhepworker\\.keys")
```

Comment:
We don't need SSLKEYSTOREWORKER, SSLKEYSTOREVWORKER, SSLKEYSTORECLIENT anymore :)

Reply:
done